modules/kmm-customizing-upgrades-for-kernel-modules.adoc

```diff
@@ -46,7 +46,7 @@ $ oc label node/<node_name> kmm.node.kubernetes.io/version-module.<module_namesp
 
 . If required, as the cluster administrator, perform any additional maintenance required on the node for the kernel module upgrade.
 +
-If no additional upgrading is needed, you can skip Steps 3 through 6 by updating the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>` label value to the new `$moduleVersion` as set in the `Module`.
+If no additional upgrading is needed, you can skip Steps 3 through 6 by updating the `kmm.node.kubernetes.io/version-module.<module.namespace>.<module.name>` label value to the new `$moduleVersion` as set in the `Module`.
```

Comment:
@cdvultur These are the only instance I found. But, as we discussed, there are instances similar to these in this and other sections, such as: oc label node/<node_name> kmm.node.kubernetes.io/version-module.**<module_namespace>.<module_name>**=<desired_version> . Should we be looking at these?

Reply:
@cdvultur We will not use periods here, we will use underscores to conform to style guide.